src/gluonts/mx/distribution/inflated_beta.py

```diff
@@ -96,9 +96,8 @@ def log_prob(self, x: Tensor) -> Tensor:
 
 
 class ZeroInflatedBeta(ZeroAndOneInflatedBeta):
-    r"""
-    Zero Inflated Beta distribution as in Raydonal Ospina, Silvia L.P. Ferrari:
-    Inflated Beta Distributions
+    r"""Zero Inflated Beta distribution as in Raydonal Ospina, Silvia L.P.
+    Ferrari: Inflated Beta Distributions.
```

Comment:
Some docstrings are changed to have a newline after the quote symbols, but this change does the opposite: is this because it's a raw string here?